File: ECOO 2017/Round 1/src/P1.py
# ECOO Round 1
# Victoria Park CI Team 1
# P1

with open("../DATA10.txt") as input:
    # string = map(str, input.read().split())
    # inputInt = map(int, input.readline().split())
    marks = [[] for i in xrange(1)]
    for i in xrange(1):
        weight = map(int, input.readline().split())
        numStudents = int(input.readline())
        for j in xrange(numStudents):
            marks[i].append(input.readline().strip())
    input.close()

print marks

File: ECOO 2017/Round 1/src/P1.cpp
// ECOO Round 1
// (c) Victoria Park CI Team 1, 2017
// Eric Li, Simon Liu, Richard(Linyin) Sun, Andrew Wang
// P1 - Munch n Brunch

#include <iostream>
#include <fstream>
#include <string>
using namespace std;

ifstream fin;

int main() {

    /* File IO */
    fin.open("DATA12.txt");

    for (int testCase = 1; testCase <= 10; testCase++) {

        int tripCost;
        fin >> tripCost;

        double p1, p2, p3, p4;
        fin >> p1 >> p2 >> p3 >> p4;

        int totalStudents;
        fin >> totalStudents;

        p1 *= totalStudents;
        p2 *= totalStudents;
        p3 *= totalStudents;
        p4 *= totalStudents;

        p1 *= 12;
        p2 *= 10;
        p3 *= 7;
        p4 *= 5;

        int funds = p1 + p2 + p3 + p4;
        funds /= 2;

        if (funds >= tripCost) {
            cout << "NO" << endl;
        } else {
            cout << "YES" << endl;
        }
    }

    return 0;
}
